frontend/src/views/SecretOverviewPage/components/SecretOverviewTableRow/SecretOverviewTableRow.tsx

```diff
@@ -81,6 +86,11 @@ export const SecretOverviewTableRow = ({
                       <FontAwesomeIcon icon={faCircle} />
                     </Tooltip>
                   )}
+                  {isOverridden && (
```

Comment:
So now how will the user's remove the override here

Reply:
Thanks for the feedback.
I think the issue was just about displaying it. It's possible to remove the override in the detailed view.

> Put the overridden secret icon in the grid of ticks and crosses where there is a secret that has been overwritten by a personal value, so that it is possible to easily identify overridden secrets.

Reply:
Hey @SMLukwiya 

The overview page has a edit functionality where user's can edit and save secret. Now even thought your displaying it, when user's edit it your not handling the case where its editing the general or the override value. 

This will make user's confused. So requesting to implement secret override CRUD like dashboard here as well please. We cannot merge without that as this is more confusing for users

Reply:
got it, looking into it.

Reply:
Hey @akhilmhdh, I noticed the behavior in V3 of the api, where deleting an override from the overview page works but reverts the secret to the original value, and deleting the original secret removes the secret from an environment all together. In the dashboard, deleting an override behaves the same as deleting the original secret. Is this difference the current expected behavior? You can view this attachment of the current approach [short video](https://share.vidyard.com/watch/hScpupPjxatqyqvGoVduZ9?)